modules/internal/ChapelHashtable.chpl

```diff
@@ -588,8 +580,8 @@ module ChapelHashtable {
             // move the key and value from the old entry into the new one
             ref dstSlot = table[newslot];
             dstSlot.status = chpl__hash_status.full;
-            _moveInit(dstSlot.key, _moveToReturn(oldEntry.key));
-            _moveInit(dstSlot.val, _moveToReturn(oldEntry.val));
+            moveInitialize(dstSlot.key, _moveToReturn(oldEntry.key));
```

Comment:
AFAIK `_moveToReturn` can be replaced by `moveToValue` from Memory.Initialization (but you could leave that for future work if you want).

Reply:
Thanks for attempting it - but just noting you might have to use static typeof in that function in Memory.Initialization too.

Reply:
Oh, actually it's probably fine, because there the argument is initialized memory.

Reply:
I worried about that too, but came to the same conclusion after testing showed that the change didn't seem to be necessary.